[Excel](Custom Functions) Updates to CFs for Shared Runtime (#1835)
OfficeDev/office-js-docs-pr: docs/excel/custom-functions-overview.md

- Uses `<ExtensionPoint>` and `<Resources>` elements that are unique to a custom functions manifest. These elements contain the information about the locations of the JavaScript, JSON, and HTML files.
- Specifies which runtime to use for your custom function. We recommend always using the shared runtime unless you have a specific need for another runtime, because the shared runtime allows for the sharing of data between functions and the task pane.

If you are using the Yo Office generator to create files, we recommend adjusting your manifest to use shared runtime, as this is not the default for these files. To change your manifest, follow the instructions in [Configure your Excel add-in to use a shared JavaScript runtime](./configure-your-add-in-to-use-a-shared-runtime.md).

To see a full working manifest from a sample add-in, see [this Github repository](https://github.com/OfficeDev/PnP-OfficeAddins/blob/master/Samples/excel-shared-runtime-global-state/manifest.xml).

[!include[manifest guidance](../includes/manifest-guidance.md)]

## Coauthoring

> Excel on the web and Windows connected to an Office 365 subscription allow you to coauthor within Excel. If your workbook uses a custom function, your colleague will be prompted to load the custom function's add-in. Once you both have loaded the add-in, the custom function will share results through coauthoring.

Reviewer: A couple small grammar nits:
```suggestion
Excel on the web and Windows connected to an Office 365 subscription allow you to coauthor in Excel. If your workbook uses a custom function, your coauthoring colleague is prompted to load the custom function's add-in. Once you both have loaded the add-in, the custom function shares results through coauthoring.
```
